tests/unit/test_export_to_nitrate.py

```diff
@@ -0,0 +1,116 @@
+import os
+import shutil
+from pathlib import Path
+import tempfile
+from tmt.export import markdown_to_html
+from unittest import TestCase
+
+TEST_DIR = Path(__file__).parent
+
+
+class NitrateExportAutomated(TestCase):
+    def setUp(self):
+        self.tmp_dir = Path(tempfile.mktemp(prefix=str(TEST_DIR)))
+        shutil.copytree(TEST_DIR, self.tmp_dir)
+        self.cwd = os.getcwd()
+        self.dir_name = 'export_case_automated'
+
+    def test_markdown_to_html_one_heading(self):
+        os.chdir(self.tmp_dir / self.dir_name)
+        files = os.listdir()
+        file_name = 'test.md'
+        self.assertIn(file_name, files)
+
+        html = markdown_to_html("Setup", file_name)
+        html_generated = """<ol>
+<li>Install tmt
+<code>bash
+sudo dnf install -y tmt-all</code></li>
+<li>Create and use a brand new git repo
+<code>bash
+cd $(mktemp -d)
+git init</code></li>
+</ol>
+"""
+        self.assertEqual(html, html_generated)
+
+    def test_markdown_to_html_two_headings(self):
+        os.chdir(self.tmp_dir / self.dir_name)
+        files = os.listdir()
+        file_name = 'test.md'
+        self.assertIn(file_name, files)
+
+        html = markdown_to_html("Step", file_name)
+        html_generated = """<p>Verify tmt shows help page
+<code>bash
+tmt --help</code></p>
+--------------------------------------------------------------------------------\
+<p>Check that error about missing metadata is sane
+<code>bash
+tmt tests ls</code></p>
+--------------------------------------------------------------------------------\
+<p>Initialize metadata structure
+<code>bash
+tmt init</code></p>
+--------------------------------------------------------------------------------\
+"""
+        self.assertEqual(html, html_generated)
+
+    def test_markdown_to_html_empty_file(self):
+        os.chdir(self.tmp_dir / self.dir_name)
+        files = os.listdir()
+        file_name = 'test_empty.md'
+        self.assertIn(file_name, files)
+        html = markdown_to_html("Step", file_name)
+        html_generated = None
+        self.assertEqual(html, html_generated)
+
+    def test_markdown_to_html_last_heading_wo_line(self):
```

Comment:
What does the `_wo_` mean?

Reply:
`_wo_` is [Acronym](https://acronyms.thefreedictionary.com/WO) for without

Reply:
But the last heading includes lines...

```
        html_generated = """<p>Optionally remove temporary \
directory created in the first step</p>
```

Am I missing something?

Reply:
It means that I wasn't clear enough. The idea was to check that there is no "long line" ('-'*80) after last section (last heading).

Reply:
Ah, I see. Thanks for clarification.